<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>391</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Menu_pg">
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sea Raider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Начать игру</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Exit_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Connect_pg">
     <widget class="QLabel" name="Name_c">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sea Raider</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Ip_line">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Port_line">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>55555</string>
      </property>
     </widget>
     <widget class="QLabel" name="IpLab">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ip адрес</string>
      </property>
     </widget>
     <widget class="QLabel" name="PortLab">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connect_button">
      <property name="geometry">
       <rect>
        <x>244</x>
        <y>250</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Присоединиться</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RegLog_pg">
     <widget class="QLabel" name="Name_rl">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sea Raider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Reg_button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Log_button">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>190</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Войти</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Lobby_pg">
     <widget class="QLabel" name="PlayersLb">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Игроки в лобби</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Quit_button">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>330</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Отсоединиться и выйти</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>321</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="WidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>332</width>
         <height>319</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Add_button">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить игрока</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="YourLb">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ваша статистика</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserLb">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="WinLb">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Побед:</string>
      </property>
     </widget>
     <widget class="QLabel" name="WinLb2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="LoseLb">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>139</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Поражений:</string>
      </property>
     </widget>
     <widget class="QLabel" name="LoseLb2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>135</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="MatchLb">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>175</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Всего матчей:</string>
      </property>
     </widget>
     <widget class="QLabel" name="MatchLb_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>170</y>
        <width>49</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Select_pg"/>
    <widget class="QWidget" name="Game_pg">
     <widget class="Line" name="centralLine">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>16</width>
        <height>381</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Exit_button</tabstop>
  <tabstop>Ip_line</tabstop>
  <tabstop>Port_line</tabstop>
  <tabstop>Reg_button</tabstop>
  <tabstop>Log_button</tabstop>
  <tabstop>Quit_button</tabstop>
  <tabstop>start_button</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>Add_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
